File: test_mail.py
import smtplib    # Importation du module
serveur = smtplib.SMTP('smtp-in.orange.fr', 587)    # Connexion au serveur sortant (en précisant son nom et son port)
#serveur.starttls()    # Spécification de la sécurisation
serveur.login("nakhila@orange.fr", "Amine06..")    # Authentification
message = "VOTRE MESSAGE"    # Message à envoyer
serveur.sendmail("nakhila@orange.fr", "nakhila@orange.fr", message)    # Envoie du message
serveur.quit()    # Déconnexion du serveur

File: test_mail.php
<?php
  include("class.phpmailer.php");
  include("class.smtp.php");
  date_default_timezone_set("Europe/Paris");
  $mail = new PHPMailer();
  $body             = "Test de PHPMailer.";
  $mail->IsSMTP();
  $mail->SMTPAuth   = true;
  $mail->SMTPOptions = array('ssl' => array('verify_peer' => false,'verify_peer_name' => false,'allow_self_signed' => true)); // ignorer l'erreur de certificat.
  $mail->Host       = "smtp-in.orange.fr";
  $mail->Port       = 587;
  $mail->Username   = "nakhila@orange.fr";
  $mail->Password   = "Amine06..";
  $mail->From       = "nakhila@orange.fr"; //adresse d’envoi correspondant au login entré précédemment
  $mail->FromName   = "Amine"; // nom qui sera affiché
  $mail->Subject    = "voici l'objet"; // sujet
  $mail->AltBody    = "corps du message au format texte"; //Body au format texte
  $mail->WordWrap   = 50; // nombre de caractères pour le retour à la ligne automatique
  $mail->MsgHTML($body);
  $mail->AddReplyTo("nakhila@orange.fr","Amine");
  //$mail->AddAttachment("./examples/images/phpmailer.gif"); pièce jointe si besoin
  $mail->AddAddress("nakhila@orange.fr");
  $mail->IsHTML(true); // envoyer au format html, passer a false si en mode texte
  if(!$mail->Send()) {
    echo "Mailer Error: " . $mail->ErrorInfo;
  } else {
    echo "Le message à bien été envoyé";
  }
?>
